### DISCUSSION TITLE
Does Zinc sync data from S3 across multiple Zinc instances

### DISCUSSION THREAD

**Discussion by deepjyoti30** [zinclabs/zinc#677] [Q&A]
I have an use-case where I want to use Zinc with S3 as storage but I have different instances of Zinc connected to the same storage and I want the data to be synced across my Zinc instances.

Is that something that Zinc already takes care of?

As an example: say I have two Zinc instances named`zinc-1` and `zinc-2`. Both of them are connected to the same s3 storage and an index is created named `example`.

If `zinc-1` indexes a document to the `example` index, will it available from `zinc-2` if the document is fetched with the ID or even searched for?

**Comment by hengfeiyang** [comment]
Fow now, zinc just can work as a single node.
